src/settings.service.js

```diff
@@ -368,12 +374,25 @@ plugin.service('wgnConfigSettings', ['$q', 'wgnConfigInputs', function ($q, conf
 		/**
 		 * Returns highlighted inputs.
 		 *
-		 * @return {Array<string>} An array of input ids.
+		 * @return {Array<string>} An array of input objects.
 		 */
 		srv.getHighlighted = function () {
 			return _highlightedFields;
 		};
 
+
+		/**
+		 * Returns all field definitions.
+		 *
+		 * @return {Array<string>} An array of objects.
+		 */
+		srv.getFields = function () {
+			return _settings.pages.map(function(page) {
+				return page.fields;
+			}).reduce((acc, val) => acc.concat(val), []);
```

Comment:
@AParks sorry but we can't use arrow functions in here! :)

We've updated mayan so that only the actual plugin source is transpiled but node_modules aren't...

Reply:
ah damn that sucks, then how come its working for me?

Reply:
@AParks because it works in newer Chrome and Firefox versions.... but not in the full set of browsers we support for clients...

Reply:
@tehpsalmist yeah ask Marc S